boards/arm/thingy91_nrf9160/thingy91_pm_static.yml

```diff
@@ -1,4 +1,4 @@
-app: {address: 0x18200, size: 0x5ae00}
+app: {address: 0x18000, size: 0x5ae00}
```

Comment:
As mentioned in the sdk-zephyr PR, update the sizes, for 'tfm_nonsecure' and 'app'.